### PR TITLE
fix(config): remove phantomjs in favor of chrome headless

### LINKED ISSUE #3088: Perform karma CI feature tests on headless chromium rather than phantomjs
One of our biggest problems in landing Pull Requests is Travis CI flakes.  headless chromium might  be just enough smaller/faster to make a difference. 

This may be a simple change:
https://medium.com/@metalex9/replace-phantomjs-with-headless-chromium-for-javascript-unit-testing-in-karma-59812e6f8ce4

### PR DESCRIPTION
**Pull request by lusarz**
fix #3088

### CODE REVIEW

**Comment by johnjbarton** [comment]
The crash during travis maybe related to https://github.com/karma-runner/karma-chrome-launcher/issues/188

**Comment by lusarz** [comment]
I fixed issue by creating [custom launcher](https://github.com/karma-runner/karma/pull/3175/commits/e1cc747e789ac781fa40928426d61bbed2816df5#diff-cf88e2e1e59fd1b5b1ebe6031fdc7b98R56) with `--no-sandbox` flag passed to chrome.

File: package.json [hunk 1]
```diff
@@ -378,6 +378,7 @@
     "mime": "^2.3.1",
     "minimatch": "^3.0.2",
     "optimist": "^0.6.1",
+    "puppeteer": "^1.9.0",
```
Comment (edmorley) [inline package.json]: @johnjbarton, hi :-) Was this meant to be added under `devDependencies` instead?

Comment (devoto13) [reply package.json]: @edmorley This is already fixed on master. See https://github.com/karma-runner/karma/issues/3191.

Comment (edmorley) [reply package.json]: Ah sorry missed that - thank you :-)

**Pull request merged**